shared/middleware/helpers/assets.rb

```diff
@@ -0,0 +1,68 @@
+#
+# Assets
+#
+class AssetBucket
+
+  ALLOWED_FILE_TYPES = %w(jpg jpeg gif png mp3)
+
+  def initialize
+    params = {region: 'us-east-1'}
```

Comment:
I believe the region should come from config rather than being hard-coded; this looks like it won't work if we start running in a new region.